rke/k8s_version_info.go

```diff
@@ -117,10 +131,24 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 		},
 		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
 		// This version includes nodelocal dns only available in RKE v1.0.7 and up
+		// It also includes ingress-nginx 0.32.0
+		"v1.17.5-rancher1-2": {
+			MinRancherVersion: "2.3.7-rc0",
+			MinRKEVersion:     "1.0.7",
+		},
+		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
+		// This version includes nodelocal dns only available in RKE v1.0.7 and up
 		"v1.18.2-rancher1-1": {
 			MinRancherVersion: "2.4.3-rc99",
```

Comment:
Can you update this to `2.4.4-rc0`? Otherwise this will be available in v2.4.3. @aiyengar2 I believe this is what it should be instead of `2.4.3-rc99`

Reply:
Will be updating this to 2.4.4-rc0 in https://github.com/rancher/kontainer-driver-metadata/pull/238

Reply:
changed